spec/json_validation_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'json'
+require 'json_schemer'
+
+describe 'json_validation' do
+  Dir['./**/*.json'].each do |json_file|
+    next if json_file.include? 'schema.json'
+
+    name = File.basename(json_file, '.json')
+    dir = File.dirname(json_file)
+    it json_file.to_s do
+      json_schema_candidates = [
+        "#{dir}/#{name}.schema.json",
+        "#{dir}/schema.json",
+      ]
+
+      json_schema_files = json_schema_candidates.select { |f| File.exist?(f) }
+      if json_schema_files.empty?
+        pending("NO SCHEMA FOR #{name} -> #{json_schema_candidates}")
+        raise
+      end
+
+      json_data = JSON.parse(File.read(json_file))
+      json_schema = JSON.parse(File.read(json_schema_files.first))
```

Comment:
Give `Pathname.new(json_schema_files.first)` and let `JSONSchemer` read the file.